src/main/kotlin/com/autonomousapps/Flags.kt

```diff
@@ -8,19 +8,14 @@ import org.gradle.api.Project
 @Suppress("DEPRECATION") // forUseAtConfigurationTime()
 object Flags {
 
-  internal const val FLAG_CLEAR_ARTIFACTS = "dependency.analysis.clear.artifacts"
```

Comment:
Would you mind leaving these flags here? (removing the methods is fine.) I'm thinking I might like to do a check at some point to let folks know about flags that no longer do anything to help them clean up their builds.

Reply:
I reverted this line.